README.md

```diff
@@ -12,7 +12,18 @@ Clone this repo, install and setup the following software tools. The versions in
 ### Requirements
 - [Xcode](https://developer.apple.com/xcode/). _(Version 10.1)_.
 - [Go](http://golang.org/doc/install). _(Version 1.12.1 tested, 1.11 should work too)_.
-  - Ensure your `$GOPATH` environment variable is set and that `$GOPATH/bin` is added to your `$PATH` environment variable.
+  - Ensure your `$GOPATH` environment variable is set 
+  - Ensure that your `$GOPATH/bin` is added to your `$PATH` environment variable.
+  - Ensure that your `GOBIN=$GOPATH/bin` is added to your `$PATH` environment variable.
```

Comment:
This should not be added to `$PATH`, it should just be set the same way `$GOPATH` is set.